docs/modelplugin.md

```diff
@@ -215,3 +209,35 @@ was needed.
 > If the generator program reports that a dependency was required e.g.
 **openconfig-inet-types.yang** then the version of this file with a date equal
 to or before 2017-07-14 should be downloaded - it is **openconfig-inet-types@2017-07-14.yang**
+
+## Readonly paths in OpenConfig models
```

Comment:
I think this is valid for all models not just openconfig.

Reply:
This is specifically an OpenConfig section of the document, so I thought it was appropriate